src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java

```
@@ -21,17 +24,18 @@ public ShoppingCart add(ShoppingCart shoppingCart) {
             transaction = session.beginTransaction();
             session.save(shoppingCart);
             transaction.commit();
-            return shoppingCart;
+            logger.info("Shopping cart " + shoppingCart + " was created.");
         } catch (Exception e) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new DataProcessingException("Can't create shopping cart " + shoppingCart, e);
+            logger.error("Can't create shopping cart " + shoppingCart, e);
```

let's throw an exception and not replace it with loggin